.github/workflows/main.yml

```diff
@@ -47,7 +47,7 @@ jobs:
         run: yarn typecheck
 
       - name: Running linters and tests
-        run: yarn run jest --projects jest.{eslint,stylelint,test,ts-test,bundle}.config.js --reporters github-actions
+        run: yarn run jest --projects jest.{eslint,test,ts-test,bundle}.config.js --reporters github-actions
```

Comment:
I disabled `stylelint` following @emmenko's  suggestion.

Since it currently is only lintin `js` files and we migrated to typescript, I will remove it from the repository as a follow up.

We can have a separate discussion about how we want to lint CSS rules after we talk about its future in `ui-kit`.